frontend/views/pages/GroupDashboard.vue

```diff
@@ -2,22 +2,28 @@
 page(pageTestName='dashboard' pageTestHeaderName='groupName' v-if='groupSettings.groupName')
   template(#title='') {{ groupSettings.groupName }}
 
-  start-inviting(v-if='groupMembersCount === 1')
+  add-income-details-widget(v-if='!hasIncomeDetails')
 
-  proposals-widget
+  template(v-if='hasIncomeDetails')
+    start-inviting(v-if='groupMembersCount === 1')
 
-  page-section(title='This months overview')
-    overview
+    contributions-widget(v-if='hasIncomeDetails')
 
-  page-section(title='July Overview')
-    //- group-pledges-graph
-    progress-overview
+  proposals-widget(v-if='hasIncomeDetails || Object.keys(currentGroupState.proposals).length > 0')
```

Comment:
Object.keys(currentGroupState.proposals).length > 0
-> should we add this logic into computed property?

Reply:
done